pkg/retry/retryer_test.go

```diff
@@ -422,86 +422,93 @@ func TestRetryWithOneTimePolicy(t *testing.T) {
 
 // test specified error to retry
 func TestSpecifiedErrorRetry(t *testing.T) {
```

Comment:
Refactor this with `t.Run(...)` to eliminate possible intersection of states between cases.

Reply:
No change of any test logic here.